ravenframework/SupervisedLearning/ARMA.py

```diff
@@ -2537,14 +2544,14 @@ class armaResultsProxy:
     Class that can be used to artifically construct ARMA information
     from pre-determined values
   """
-  def __init__(self, arparams, maparams, sigma):
+  def __init__(self, polynomial_ar, polynomial_ma, sigma):
```

Comment:
I approved too soon. These variable names need to follow camelCase conventions or can be something like `polyar`.

Reply:
Ah sorry, always forget about that one. Just pushed a fix.